runtime/tracing/tracing-core/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+description = "Common tracing classes"
+extra["displayName"] = "Smithy :: Kotlin :: Tracing :: Core"
+extra["moduleName"] = "aws.smithy.kotlin.runtime.tracing"
+
+val kotlinLoggingVersion: String by project
+
+kotlin {
+    sourceSets {
+        commonMain {
+            dependencies {
+                // Necessary for Instant
+                api(project(":runtime:runtime-core"))
+
+                // Necessary for Closeable
+                api(project(":runtime:io"))
+
+                // Necessary for InternalApi
+                api(project(":runtime:utils"))
+
+                // Necessary for Logger
+                api(project(":runtime:logging"))
+
+                implementation("io.github.microutils:kotlin-logging:$kotlinLoggingVersion")
```

Comment:
question: Why is this necessary? Isn't this hidden as an implementation detail of `:runtime:logging`?

Reply:
No, the `KotlinLoggingTraceProbe` accesses `mu.KLogger` and `mu.KotlinLogging` directly and doesn't go through **:runtime:logging**'s `KotlinLoggingAdapter`. We could _make_ it do that but it seems unnecessary given that we [plan to remove `KotlinLoggingAdapter`](https://github.com/awslabs/aws-sdk-kotlin/issues/757) shortly. At that time we'll probably move (or remove) `Logger` too and then **:runtime:logging** itself will be removed.

Reply:
We should get this into the backlog and track as part of GA